frontend/index.html

```diff
@@ -11,138 +11,157 @@
     <header class="main-theme">
         <div class="account container">
             <ul class = account-links>
-                <li><a href="#account"><img class="user" src="./assets/images/person-fill.svg" alt="">Account</a></li>
-                <li><a href="#contactus">Contact us</a></li>
+                <li><a href="#account"><img class="user" src="./assets/images/person-fill.svg" alt=""><span>Account</span></a></li>
+                <li><a href="#contactus"><span>Contact us</span></a></li>
             </ul>
         </div>
 
         <div class="menu container">
-            <div class="menu__item">
+
+            <div onclick=showSideBar() class=" menu-button ideas">
+                <a href="#hamburger">
```

Comment:
Can we make this element a `button` instead? https://medium.com/@flik185/the-semantics-of-a-click-2043c85b8195

This would also prevent getting the `#hamburger` and `#close` in the url that we're currently having. 

Also I would attach the click event to the button rather than the `div` - when using a screen reader for example, and navigating to this element with the keyboard or with the accessibility tools on a device, the screenreader would recognise the button element as clickable.

Reply:
hamburger menu buttons changed from anchor tag to button